src/main/java/net/snowflake/ingest/streaming/internal/FlushService.java

```diff
@@ -360,6 +361,9 @@ void distributeFlushTasks() {
     List<Pair<BlobData<T>, CompletableFuture<BlobMetadata>>> blobs = new ArrayList<>();
     List<ChannelData<T>> leftoverChannelsDataPerTable = new ArrayList<>();
 
+    // The API states that the number of available processors reported can change and therefore, we
+    // should poll it occasionally.
```

Comment:
Out of curiosity under what circumstances can this change?

Reply:
I had the same question and looked around a bit but could not find any concrete answers. So this is mostly defensive in case in the future there is a use-case for this.